lib/clusteringBrowser.js

```diff
@@ -0,0 +1,18 @@
+// This is used in browsers only and is designed to allow the rest of
```

Comment:
This might work since we can't really test it, can we?

```suggestion
/* istanbul ignore file */
// This is used in browsers only and is designed to allow the rest of
```

Reply:
That's a great suggestion, thank you!